### PR TITLE
Add some missing using statements.

### LINKED ISSUE #4143: Fix TFS integration for WebSockets.Client
In https://github.com/dotnet/corefx/blob/master/src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.WinRT.cs:

```
// TODO: ** TFS BUILD IS BROKEN **  
                // This doesn't compile right now due to missing types 'WebErrorStatus' and 'RTWebSocketError' 
                 // Commenting out for now to allow the build to resume. 
                 //WebErrorStatus status = RTWebSocketError.GetStatus(ex.HResult); 
                 //var inner = new Exception(status.ToString(), ex); 
```

### PR DESCRIPTION
**Pull request by pgavlin**
The lack of these statements was breaking the TFS build.

Fixes #4143.

### CODE REVIEW

**Comment by pgavlin** [comment]
@davidsh @mconnew PTAL. I'm validating the TFS build now.

**Comment by pgavlin** [comment]
(with some help from @ellismg :))

File: src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.WinRT.cs [hunk 1]
```diff
@@ -41,11 +44,8 @@ private WebSocketHandle(WinRTWebSocket webSocket)
...
             catch (Exception ex)
             {
-                // TODO #4143: ** TFS BUILD IS BROKEN ** 
-                // This doesn't compile right now due to missing types 'WebErrorStatus' and 'RTWebSocketError'
-                // Commenting out for now to allow the build to resume.
-                //WebErrorStatus status = RTWebSocketError.GetStatus(ex.HResult);
-                //var inner = new Exception(status.ToString(), ex);
+                WebErrorStatus status = RTWebSocketError.GetStatus(ex.HResult);
+                var inner = new Exception(status.ToString(), ex);
                 var inner = ex;
```
Comment (stephentoub) [inline src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.WinRT.cs]: You also need to delete this ```var inner = ex;``` line, right?

Comment (pgavlin) [inline src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.WinRT.cs]: Ah, good catch--I thought I had. Thanks.

**Comment by stephentoub** [comment]
I don't think this will build as part of the CI, so if it builds for you successfully locally, LGTM.

**Comment by davidsh** [comment]
LGTM

**Comment by ellismg** [comment]
LGTM.  It built fine internally, modulo some existing breaks in other projects we already know about.

**Pull request merged**